buildSrc/src/main/java/DepConfig.kt

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This diff is _so_ bad!
I renamed the file, turned `DepConfig` into a class and changed all the `const val` to just `val` (since classes can't have `const val`.
There _aren't_ actually that many changes in this file :)